substratevm/src/com.oracle.svm.core.posix/src/com/oracle/svm/core/posix/JavaUtilZipSubstitutions.java

```diff
@@ -336,26 +418,42 @@ private static long init(boolean nowrap) {
         }
     }
 
-    // Inflater.c-Java_java_util_zip_Inflater_setDictionary(JNIEnv *env, jclass cls, jlong addr,
+    // (JDK 11)src/java.base/share/native/libzip/Inflater.c-static void doSetDictionary(JNIEnv *env, jlong addr, jbyte *buf, jint len)
+    private static void doSetDictionary(final long addr, final CCharPointer bytes, final int len) {
```

Comment:
We have checkstyle rules that enforce that method arguments are `final`.  (I had to be weaned from declaring them `finel` too.)

Reply:
OK I'll fix that.  Also I just figured out that mx has a checkstyle action, so I'll run that locally to verify.